### ISSUE TITLE
mouse data

### ISSUE THREAD

**Issue by Semple-bio** [saezlab/transcriptutorial#17]
Good day.
It seems most of the pipeline is suitable for mouse data, except for the step using the "progenyMembers.RData" to prepare the input files for Carnival. Is there a mouse equivalent somewhere in the package?

**Comment by adugourd** [comment]
No actually, carnival doesn't have a specific mouse implementation. However, you may simply convert your mouse gene names to human genes names and assume that the orthology between mouse and human is good enough to use the ressources across the two species.

**Comment by Semple-bio** [comment]
While that is indeed what I did, there is the problem at progenyMembers$gene$Trail
From the 2 genes associated with it, one is human-specific, so that might quite distort the calculations.

**Comment by adugourd** [comment]
Indeed, Trail is the pathway that works the worst when converting between human and mouse. 

As a general rule, always look into the actual signature of the pathway to interpret the score. So you should be fine it seems :)
